### ISSUE TITLE
Fix bug (?) in route_flow_dn: "no module named cfuncs"

### ISSUE THREAD

**Issue by gregtucker** [landlab/landlab#116]
from landlab.components.flow_routing.route_flow_dn import FlowRouter

fr = FlowRouter(rg)

fr.route_flow()
Traceback (most recent call last):

  File "<ipython-input-25-81520d145347>", line 1, in <module>
    fr.route_flow()

  File "/Users/gtucker/Dev/Landlab/landlab/landlab/components/flow_routing/route_flow_dn.py", line 233, in route_flow
    use_weave=self.weave_flag)

  File "/Users/gtucker/Dev/Landlab/landlab/landlab/components/flow_routing/flow_direction_DN.py", line 202, in flow_directions
    from .cfuncs import adjust_flow_receivers

ImportError: No module named cfuncs

**Comment by SiccarPoint** [comment]
If I remember correctly (from talking to Eric when I think this happened to me), this is a compiler problem that is likely to only affect the core dev team. I think basically the dev install isn't clever enough to realize on its own there's now Cython in the repo.

TL;DR: Run python setup.py develop again and it should just work.

Eric may correct me if I'm wrong...?

**Comment by mcflugen** [comment]
@SiccarPoint, I think you are exactly right. The extension libraries probably need to be build. This is only an issue for folks that have a development version of landlab that was installed before the Cython code was introduced into master.

**Comment by SiccarPoint** [comment]
This issue should probably be publicised to the group...

**Comment by awickert** [comment]
@SiccarPoint: I just had this same issue, and this post got me out of it. Would definitely support publicizing it somewhere.
